src/App.tsx

```diff
@@ -76,18 +63,14 @@ export const App: React.FC = () => {
                   </div>
                 )}
 
-                {author && loaded && !hasError && posts.length === 0 && (
+                {author && !loading && !hasError && posts.length === 0 && (
                   <div className="notification is-warning" data-cy="NoPostsYet">
                     No posts yet
                   </div>
                 )}
 
-                {author && loaded && !hasError && posts.length > 0 && (
-                  <PostsList
-                    posts={posts}
-                    selectedPostId={selectedPost?.id}
-                    onPostSelected={setSelectedPost}
-                  />
+                {author && !loading && !hasError && posts.length > 0 && (
```

Comment:
```suggestion
                {author && !loading && !hasError && !!posts.length && (
```

Reply:
let's move all these conditions to variables in order to keep jsx clean.
something like `isPostListVisible`, `isLoaderVisible`, etc.